client/blocks/author-selector/switcher-shell.jsx

```diff
@@ -107,7 +97,8 @@ class AuthorSwitcherShell extends React.Component {
 							onSearch={ this.onSearch }
 							placeholder={ this.props.translate( 'Find Author…', { context: 'search label' } ) }
 							delaySearch={ true }
-							ref={ this.authorSelectorSearchRef }
+							// eslint-disable-next-line jsx-a11y/no-autofocus
```

Comment:
@diegohaz What do you think of this eslint rule (`jsx-a11y/no-autofocus`). Autofocus in direct response to a user event like this shouldn’t raise a11y concerns, right? Do you think this eslint rule provides any value generally?